shopify/product-reviews-template/api/models/shopifyShop/schema.gadget.ts

```diff
@@ -0,0 +1,103 @@
+import type { GadgetModel } from "gadget-server";
+
+// This file describes the schema for the "shopifyShop" model, go to https://product-reviews-template.gadget.app/edit to view/edit your model in Gadget
+// For more information on how to update this file http://docs.gadget.dev
+
+export const schema: GadgetModel = {
+  type: "gadget/model-schema/v1",
+  storageKey: "DataModel-Shopify-Shop",
+  fields: {
+    daysUntilReviewRequest: {
```

Comment:
It seems like we should have a frontend input that allows merchants to manage this value. Also, a way to make it possible for devs to test right away

Reply:
Mo told me not to put one to start

Reply:
I can talk about it in ways to extend